src/components/buttons/base-button-wrapper/base-button-wrapper.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Spacings from '../../../materials/spacings';
+import styles from './base-button-wrapper.mod.css';
+
+// This holds the logic of the `button` element, which wraps a `span` (defined as `BaseButtonContent`) with styles
+class BaseButtonWrapper extends React.Component {
```

Comment:
Why is this called a wrapper? A wrapper is usually around something. This seems more like a base element.